core/src/sim.c

```diff
@@ -210,24 +212,6 @@ void ignore_play(SimmedPlay *sp, int lock) {
 }
 
 int handle_potential_stopping_condition(Simmer *simmer) {
-  if (simmer->num_simmed_plays < 2) {
-    return 1; // should stop
-  }
-  if (simmer->iteration_count >
-      (MAX_STOPPING_ITERATION_CT +
-       (simmer->max_plies * PER_PLY_STOPPING_SCALING))) {
```

Comment:
since we removed this, is your thought that # iterations should just be a fixed per-invocation thing?

Reply:
Yes, unless the user specified to run the sim indefinitely.